test-automation/app/accessibility_tasks.py

```diff
@@ -0,0 +1,85 @@
+from aloe import world
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.select import Select
+from selenium.webdriver.support.wait import WebDriverWait
+
+# Do not remove this import. Without it the hook will not be ran
+from app.hooks import with_firefox
+from app.selectors import *
+
+# The time the WebDriver will wait for various openstions- find element, wait for page, expected conditions
+webDriverWaitInSeconds = 5
+
+# Tries to go to the index page. If the user is not logged in, the methods logs in
+def go_to_index():
+    user_name = "demo@geri.life"
+    password = "demo123"
+
+    world.browser.get(geri_life_url)
+
+    #TODO wait for page to load
```

Comment:
How oftenwill the tests fail without this?

Reply:
This code is ran before each scenario. Without it, the tests will not run at all. Is there a problem with it? I can find a workaround.

Reply:
Well, I am hoping there is a more declarative API for waiting for page elements to become visible, or at least have an [implicit wait](http://docs.seleniumhq.org/docs/04_webdriver_advanced.jsp).

Reply:
Maybe my comment was misleading.
webDriverWaitInSeconds is the time used for implicit wait. We do not wait 5 sec each time we need to find an element. 5 sec is the maximum time the driver waits. The elements and pages are found using Expected conditions:
element = WebDriverWait(world.browser, webDriverWaitInSeconds).until(
EC.element_to_be_clickable((By.CSS_SELECTOR, selector)))
In most cases, the the test don't need to wait at all. The existing 12 tests' total run time is 3 min.